packages/ux-capture/src/Zone.js

```diff
@@ -19,7 +39,36 @@ function Zone(props) {
 	this.measureName = this.props.name;
 
 	// Create a new `ExpectedMark` for each mark
```

Comment:
we already have this comment on line 70 below

Reply:
yeah, it's not correct on this line and better suited for line 70 where expected marks are actually created.